Make sure the parent endpoint from existing first ENTRY span (#129)
apache/skywalking-java: CHANGES.md

* The namespace and cluster would be reported as instance properties, keys are `namespace` and `cluster`. Notice, if
  instance_properties_json includes these two keys, they would be overrided by the agent core.
* [Breaking Change] Remove the namespace from `cross process propagation` key.

> * Make sure the parent endpoint in tracing context from existing first ENTRY span, rather than first span only.

Reviewer: I feel tough to understand the meaning of this sentence without reading the codes.

```suggestion
* Make sure the parent endpoint in tracing context is the first ENTRY span, rather than first span only.
```

Author: `is from` is not correct, because this trace context could only have local/exit spans.
